requirements.txt

```diff
@@ -2,27 +2,27 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file=requirements.txt requirements.in
+#    pip-compile
 #
-boto3==1.13.12            # via -r requirements.in
-botocore==1.16.12         # via boto3, s3transfer
+boto3==1.14.26            # via -r requirements.in
+botocore==1.17.26         # via boto3, s3transfer
 cachecontrol==0.12.6      # via -r requirements.in
-certifi==2019.11.28       # via requests
+certifi==2020.6.20        # via requests
 chardet==3.0.4            # via requests
-colorlog==4.1.0           # via -r requirements.in
+colorlog==4.2.1           # via -r requirements.in
 contextlib2==0.6.0.post1  # via schema
 docutils==0.15.2          # via botocore
-idna==2.8                 # via requests
+idna==2.10                # via requests
 jinja2==2.11.2            # via -r requirements.in
 jmespath==0.10.0          # via boto3, botocore
 lockfile==0.12.2          # via -r requirements.in
 markupsafe==1.1.1         # via jinja2
 msgpack==1.0.0            # via cachecontrol
 python-dateutil==2.8.1    # via botocore
 pyyaml==5.3.1             # via -r requirements.in
-requests==2.22.0          # via -r requirements.in, cachecontrol
+requests==2.24.0          # via -r requirements.in, cachecontrol
```

Comment:
This will conflict with `requirements-azure.txt`

Reply:
how can we generate that file with all reqs ? (i followed your documentation)